From f9b1d7f50d8e6b9421f5ca7a0d413b5683bc3f61 Mon Sep 17 00:00:00 2001
From: Alexander Bulgak <aleksandarbulgak@yandex.by>
Date: Wed, 14 Oct 2020 22:47:12 +0300
Subject: [PATCH] mod2

---
 1.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/1.txt b/1.txt
index 485d69a..ef9a776 100644
--- a/1.txt
+++ b/1.txt
@@ -5,3 +5,5 @@ lines
 
 new lines 1
 new lines 1
+new line 2
+new line 2
-- 
2.21.0.windows.1

