From dc699564484daf5083601ea0cce5788fc48456e1 Mon Sep 17 00:00:00 2001
From: Alexander Bulgak <aleksandarbulgak@yandex.by>
Date: Wed, 14 Oct 2020 22:44:56 +0300
Subject: [PATCH 1/2] 1mod

---
 1.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/1.txt b/1.txt
index ed35c19..485d69a 100644
--- a/1.txt
+++ b/1.txt
@@ -2,3 +2,6 @@ modify two
 lines
 modify another two
 lines
+
+new lines 1
+new lines 1
-- 
2.21.0.windows.1

