From d0a4d4cf3e23e3e55dcd7207cc7c7afb55c30d74 Mon Sep 17 00:00:00 2001
From: Alexander Bulgak <aleksandarbulgak@yandex.by>
Date: Wed, 14 Oct 2020 23:32:03 +0300
Subject: [PATCH 1/2] lll1

---
 1.txt | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 1.txt

diff --git a/1.txt b/1.txt
new file mode 100644
index 0000000..c0d0fb4
--- /dev/null
+++ b/1.txt
@@ -0,0 +1,2 @@
+line1
+line2
-- 
2.21.0.windows.1

